README.md

```diff
@@ -18,22 +18,18 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
 
 | Field Name              | Type                | Description |
 | ----------------------- | ------------------- | ----------- |
-| classification:classed  | `Classed`           | **REQUIRED**. Classes in the dataset |
+| classification:class_list  | `[string]`           | **REQUIRED**. Summary of class names in the dataset |
+
+| Field Name              | Type                | Description |
```

Comment:
I'd suggest to merge the three tables :-)

Reply:
yes, good idea!